.hound.yml

```diff
@@ -4,5 +4,5 @@ scss:
 
 eslint:
   enabled: true
-  config_file: .eslintrc
+  config_file: .eslintrc.json
```

Comment:
@emekafredy You can revert the changes to this file. So it doesn't cause conflicts with what we have in the develop branch.

Reply:
Okay, thanks. I'm on it